generator/route/instance.js.hbs

```diff
@@ -0,0 +1,22 @@
+
+const container = require('src/container') // we have to get the DI
+const { get, post } = require('src/app/{{camelCase name}}')
+
+module.exports = () => {
+  const { {{#if InjectRepository}} repository: {
+    {{camelCase repositoryName}}
+  },   {{/if}} jwt } = container.cradle
+
+  const getUseCase = get({ {{#if InjectRepository}} {{camelCase repositoryName}},{{/if}} logger})
+  const postUseCase = post({  {{#if InjectRepository}} {{camelCase repositoryName}},{{/if}} logger})
+  const putUseCase = put({ {{#if InjectRepository}} {{camelCase repositoryName}},{{/if}} logger })
+  const deleteUseCase = remove({ {{#if InjectRepository}} {{camelCase repositoryName}},{{/if}} logger })
```

Comment:
I think we don't need to include `logger` here by default.

Reply:
ok

Reply:
I have found myself injecting logger in app layer in multiple scenarios. Can you expand/elaborate the reasoning for not having logger here

Reply:
Well I don't have hard reason. We can import it just add `logger` from `container.cradle` as well.